libecl/src/ecl_grid.c

```diff
@@ -500,30 +500,46 @@ which splits the lower face along the 0-3 diagnoal and the upper face
 along the 5-6 diagonal.
 */
 
-static const int tetrahedron_permutations[2][12][3] = {{{0,1,2},
+static const int tetrahedron_permutations[2][12][3] = {{
+                                                        // Top
```

Comment:
Instead of 'top / bottom / north / south / ...' you should use `I+, I-, J+, J-, K+`and `K-` - those are *reasonably* well defined.  See [this](https://github.com/Ensembles/ert/blob/master/libecl/src/ecl_grid.c#L449)  for the numbering of corners; also need to define how e.g. `{0,1,2}` should be related to a direction - it is not obvious.

Alternatively the strings can just be removed again!